docker-compose-files/docker-compose.yml

```diff
@@ -293,10 +293,12 @@ services:
     ############################################################################
 
     # grafana runs as root in latest version. *sigh*
+     # Polystat plugin install doesnt work with prior versions of Grafana. Grafana 9.0.1 is just released on 2022-06-21
     container_name: telemetry-reference-tools-grafana
-    image: grafana/grafana:latest
+    image: grafana/grafana:9.0.1
```

Comment:
hardcoding a specific version means we have to keep this up to date if there are CVE updates

Reply:
polystat plugin didnt work with latest, will work with 9.0.1. will check what the state is now by putting latest.